client_it_test.go

```diff
@@ -248,77 +242,39 @@ func TestClusterReconnection_ShutdownCluster(t *testing.T) {
 	mu := &sync.Mutex{}
 	events := []hz.LifecycleState{}
 	config := cls.DefaultConfig()
+	disconnectedWaitGroup := sync.WaitGroup{}
```

Comment:
I would like to emphasize why I am using these. In `dwg` , there is no way a reader to guess what `d` stands for. 
I think in GO, the rule of thumb is short and `descriptive`. Not just short. 
And the rule is mostly for method receivers. Since I have multipe wait groups I think it is important to emphasize the reason with the name. 
My references(Top results I google `short names in go`)
https://github.com/golang/go/wiki/CodeReviewComments#variable-names
https://betterprogramming.pub/naming-conventions-in-go-short-but-descriptive-1fa7c6d2f32a

Reply:
The example they give is: "Prefer c to lineCount" :)  And in the next paragraph: "The basic rule: the further from its declaration that a name is used, the more descriptive the name must be"

Reply:
I have changed the names to connectedWg and discoonectedWg. I think it is short enough :)